spec/kahuna_client/client/adaptive_campaign_spec.rb

```diff
@@ -0,0 +1,47 @@
+require File.expand_path('../../../spec_helper', __FILE__)
```

Comment:
i think in current versions of rspec that the `/spec` folder is automatically in load path, so `require 'spec_helper'` ought to work